pkg/inventory/storage.go

```diff
@@ -35,6 +37,12 @@ type Storage interface {
 	Store(objs object.ObjMetadataSet, status []actuation.ObjectStatus) error
 	// GetObject returns the object that stores the inventory
 	GetObject() (*unstructured.Unstructured, error)
+	// Apply applies the inventory object. This utility function is used
+	// in InventoryClient.Merge.
+	Apply(dynamic.Interface, meta.RESTMapper, StatusPolicy) error
+	// ApplyWithPrune applies the inventory object with a set of pruneIDs of
+	// objects to be pruned. This function is used in InventoryClient.Replace.
+	ApplyWithPrune(dynamic.Interface, meta.RESTMapper, StatusPolicy, object.ObjMetadataSet) error
```

Comment:
Please add a comment for the argument `object.ObjMetadataSet`.

Reply:
Added.